library/entropy_poll.c

```diff
@@ -54,28 +54,41 @@
 #define _WIN32_WINNT 0x0400
 #endif
 #include <windows.h>
-#include <wincrypt.h>
+#include <bcrypt.h>
+#if _MSC_VER <= 1600
```

Comment:
This and the identical line below should be `#if defined(_MSC_VER) && _MSC_VER <= 1600` so that it doesn't break other environments that don't support those `#pragma`.